clangd/index/ClangdIndexerImpl.cpp

```diff
@@ -150,10 +180,38 @@ class TranslationUnitToBeIndexedCollector {
       if (!Status)
         continue;
       llvm::sys::fs::file_type Type = Status->type();
+
+      bool Found = false;
+      if (!ExclusionList.empty()) {
+        unsigned Index = 0;
+        while (!Found && Index < ExclusionList.size()) {
+          if (llvm::sys::path::is_absolute(ExclusionList[Index]) &&
+              ExclusionList[Index].find(RootPath) == 0) {
+            std::string EndOfPath = ExclusionList[Index].substr(
+                RootPath.length(), ExclusionList[Index].length() - 1);
+            std::string MergePath = RootPath + EndOfPath;
+            if (Folder == MergePath)
+              Found = true;
+
+          } else {
+            std::string MergePath = RootPath + "/" + ExclusionList[Index];
+            if (Folder == MergePath)
+              Found = true;
+          }
+          Index++;
+        }
+      }
+
+      if (Found) {
+        llvm::errs() << " Folder is excluded from indexing. "
```

Comment:
It's weird here because we don't know which folder is excluded. See other comment that adds the logging at the beginning of the method.